params/version.go

```diff
@@ -65,3 +70,76 @@ func VersionWithCommit(gitCommit, gitDate string) string {
 	}
 	return vsn
 }
+
+// RuntimeInfo returns build and platform information about the current binary.
+//
+// If the package that is currently executing is a prefixed by our go-ethereum
+// module path, it will print out commit and date VCS information. Otherwise,
+// it will assume it's imported by a third-party and will return the imported
+// version and whether it was replaced by another module.
+func RuntimeInfo() string {
```

Comment:
I think this code should be in package core, where the bad block report also is. If you don't want to add more code to core/blockchain.go, just add a new file core/badblock.go and add it there. We are intentionally keeping the amount of 'actual code' in package params to a minimum, because the purpose of this package is keeping the Ethereum protocol constants.

Reply:
Makes sense! I've updated the PR to move that out of params.